src/web/src/components/NavBar/NavBar.tsx

```diff
@@ -1,10 +1,11 @@
 import Link from 'next/link';
-import { makeStyles } from '@material-ui/core/styles';
 import { AppBar, Toolbar } from '@material-ui/core';
+import { makeStyles, useTheme } from '@material-ui/core/styles';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
 import SearchIcon from '@material-ui/icons/Search';
 import HomeIcon from '@material-ui/icons/Home';
 import ContactSupportIcon from '@material-ui/icons/ContactSupport';
-import HeaderButton, { HeaderIcon } from './HeaderButton';
+import HeaderButton, { HeaderIcon } from './NavBarButton';
```

Comment:
Since we are using `NavBar`, let's use `NavBarButton` and `NavBarIcon` as well.
Sorry for only cath this now.